test/performance/Benchmarks.md

```diff
@@ -0,0 +1,166 @@
+# Benchmarks
+
+Knative performance benchmarks are tests geared towards producing useful
+performance metrics of the knative system. All the raw metrics are stored in
+[mako](https://github.com/google/mako)
+
+## Writing new benchmarks
+
+For creating new benchmarks, follow the steps:
+
+1. Create a new directory under `./test/performance/`.
+2. Create two benchmarks using in the bechmark directory using
+   [mako](https://github.com/google/mako/blob/github-push-test-1/docs/GUIDE.md#preparing-your-benchmark)
+   as mentioned in [benchmark configs](#benchmark-configs).
+3. Create a `kodata` directory and add the [four symlinks](#Benchmark-Symlinks).
+4. Write a go program that runs the test and stores the result in
+   [mako](##Writing-to-mako)
+5. Create a [setup.yaml](https://github.com/knative/serving/blob/master/test/performance/dataplane-probe/dataplane-probe-setup.yaml)
```

Comment:
I found this to be optional for my control plane test: https://github.com/knative/serving/pull/5267